autoarray/structures/grids.py

```diff
@@ -1838,3 +2123,65 @@ def wrapper(profile, grid, *args, **kwargs):
         return func(profile, grid, *args, **kwargs)
 
     return wrapper
+
+
+class MaskedGrid(AbstractGrid):
+    @classmethod
+    def manual_1d(cls, grid, mask, store_in_1d=True):
+
+        if type(grid) is list:
```

Comment:
prefer isinstance (unless you have some very specific reason here which I guess you might)

Reply:
Done